www/viewgame

```diff
@@ -1451,15 +1453,18 @@ function updateUnwishList(id, stat)
 <details><summary>Tips</summary>
 <ul>
 <li>Vote only if you've played the game.</li>
-<li>Vote on how long it would take to get to one final ending without hints or a walkthrough.</li>
+<li>Vote on how long it takes to get to one final ending without relying extensively on a walkthrough.</li>
 <li>Time spent away from the game doesn't count.</li>
+<li>To help players gauge how long a game with puzzles might take, you can note if your time is with or without hints.</li>
 <li>Times don't need to be exact. Good-faith estimates are fine.</li>
 <li>Your vote will be publicly attributed to you.</li>
 </ul></details>
 <p><label for='hours'>Hours:</label>
 <input type='number' name='hours' id='hours' min=0 max=200>
 <label for='minutes'>Minutes:</label>
 <input type='number' name='minutes' id='minutes' min=0 max=59></p>
+<p><label for='timeNote'>Time Note:</label>
+<input type='text' name='timeNote' id='timeNote' maxlength='150' size='21' placeholder='Without hints.'></p>
```

Comment:
This field is weird, so I like placeholder text to include some text that makes it clearly an example.

In https://github.com/iftechfoundation/ifdb/pull/1153#discussion_r1848895677 I suggested `placeholder='e.g. "Completionist" or "No hints"'`. It'll fit if we crank `size` up from 21 to 30.

Other ideas:
* `placeholder="With/without hints"`
* `placeholder="I used a walkthrough..."`

Reply:
FWIW, this is a subjective thing… people can disagree about what's good in visual design.

If we want to line up the labels for "Time Note" and "Hours", we can use a `<table>` or a CSS grid. `<table>` is easier to learn, but less flexible; CSS grid can be quite tricky to learn, but, once you learn it, it's quite handy.

Reply:
I changed the placeholder text to say "With/without hints." and it fits. Do you still want to make the field bigger?

Reply:
I think we're good